utilities/test_suite/HIP/runTests.py

```diff
@@ -38,6 +57,7 @@ def rpp_test_suite_parser_and_validator():
     parser.add_argument('--profiling', type = str , default='NO', help='Run with profiler? - (YES/NO)', required=False)
```

Comment:
Please ensure these profiler times reported with this flag --profiling YES also are similar to those from previous test suite